packages/cfpb-atomic-component/src/utilities/transition/AlphaTransition.js

```diff
@@ -1,12 +1,13 @@
 // Required modules.
 import BaseTransition from './BaseTransition.js';
-import EventObserver from '../../mixins/EventObserver.js';
+import EventObserver from '@cfpb/cfpb-atomic-component/src/mixins/EventObserver.js';
 
 // Exported constants.
 const CLASSES = {
-  BASE_CLASS: 'u-alpha-transition',
-  ALPHA_100:  'u-alpha-100',
-  ALPHA_0:    'u-alpha-0'
+  CSS_PROPERTY: 'opacity',
```

Comment:
@contolini 

This addition is a bummer because it has to match the CSS that gets transitioned in `u-alpha-transition`. I discovered that the fade is not working on the feedback form on https://www.consumerfinance.gov/ask-cfpb/what-is-a-debt-to-income-ratio-why-is-the-43-debt-to-income-ratio-important-en-1791/ and the reason is the `background-color` is being transitioned there for some reason, which fires the `transitionEnd` on the target element before the fade has been performed. The change in this PR of passing in the CSS property that is being transitioned was my solution for this, which is then checked in the `_transitionComplete` handler. Let me know if you see a better way.

Reply:
That makes sense. FWIW it is nice knowing which property the transition modifies without having to hunt down the applicable CSS file.